From 0a5a5a4230c4ad0641dda980e746289c4ddd1c93 Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Tue, 19 Aug 2014 15:26:45 -0400
Subject: [tools] perf: Add missing data.h into LIB_H headers

Message-id: <1408462094-14194-237-git-send-email-jolsa@redhat.com>
Patchwork-id: 88225
O-Subject: [PATCH RHEL7.1 BZ1131394 236/325] perf tools: Add missing data.h into LIB_H headers
Bugzilla: 1131394
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1131394
https://bugzilla.redhat.com/show_bug.cgi?id=1131394

upstream
========
commit 6e6dc401d528e3b64626de82322fa237f1c1e576
Author: Jiri Olsa <jolsa@redhat.com>
Date: Sat Oct 26 20:53:14 2013 +0200

description
===========
Adding missing data.h into LIB_H headers so the build could keep up with
its changes.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/Makefile.perf b/tools/perf/Makefile.perf
index 8a9ca38..bc7cfa1 100644
--- a/tools/perf/Makefile.perf
+++ b/tools/perf/Makefile.perf
@@ -295,6 +295,7 @@ LIB_H += ui/helpline.h
 LIB_H += ui/progress.h
 LIB_H += ui/util.h
 LIB_H += ui/ui.h
+LIB_H += util/data.h
 
 LIB_OBJS += $(OUTPUT)util/abspath.o
 LIB_OBJS += $(OUTPUT)util/alias.o
-- 
1.7.1